Gemfile

```diff
@@ -1,5 +1,5 @@
 source 'http://rubygems.org'
 
-ruby '1.9.3', engine: 'jruby', engine_version: '1.7.4'
+ruby '1.9.3'  # , engine: 'jruby', engine_version: '1.7.4'
```

Comment:
Oops. I committed that accidentally. 

Was trying to run locally on my laptop since I don't have a Heroku account handy. I'll remove.